graphql/schema/types/scene.graphql

```diff
@@ -41,6 +42,7 @@ type Scene {
   phash: String
   interactive: Boolean!
   interactive_speed: Int
+  captions: String
```

Comment:
This should be a `[String!]`. It's probably overkill to store captions in the database in a string list association table, but I think we should at least be performing the string splitting on the server end rather than requiring the client to do it.